app/models/project.rb

```diff
@@ -140,4 +142,8 @@ def destroy
     samples.each(&:destroy)
     super
   end
+
+  def add_default_metadata_fields
```

Comment:
There's a place in projects_controller `create` method where this is still being done. Can you remove that?

Reply:
👍